Shared/Samples/Choose camera controller/ChooseCameraControllerView.swift

```diff
@@ -0,0 +1,161 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import ArcGISToolkit
+import SwiftUI
+
+struct ChooseCameraControllerView: View {
+    /// A scene with imagery basemap style and a tiled elevation source.
+    @State private var scene: ArcGIS.Scene = {
+        // Creates a scene.
+        let scene = Scene(basemapStyle: .arcGISImageryStandard)
+        
+        // Sets the initial viewpoint of the scene.
+        scene.initialViewpoint = Viewpoint(
+            latitude: .nan,
+            longitude: .nan,
+            scale: .nan,
+            camera: Camera(
+                lookingAt: Point(
+                    x: -109.937516,
+                    y: 38.456714,
+                    spatialReference: .wgs84
+                ),
+                distance: 5500,
+                heading: 150,
+                pitch: 20,
+                roll: 0
+            )
+        )
+        
+        // Creates a surface.
+        let surface = Surface()
+        
+        // Creates a tiled elevation source.
+        let worldElevationServiceURL = URL(
+            string: "https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"
+        )!
+        let elevationSource = ArcGISTiledElevationSource(url: worldElevationServiceURL)
+        
+        // Adds the elevation source to the surface.
+        surface.addElevationSource(elevationSource)
+        
+        // Sets the surface to the scene's base surface.
+        scene.baseSurface = surface
+        return scene
+    }()
+    
+    /// A graphics overlay containing an airplane graphic.
+    private static let graphicsOverlay: GraphicsOverlay = {
+        let graphicsOverlay = GraphicsOverlay(graphics: [Graphic.airplane])
+        graphicsOverlay.sceneProperties.surfacePlacement = .absolute
+        return graphicsOverlay
+    }()
+    
+    /// A Boolean value indicating whether the settings view should be presented.
+    @State var isShowingSettings = false
+    
+    /// The sun lighting mode of the scene view.
+    @State private var cameraControllerKind: CameraControllerKind = .globe
+    
+    enum CameraControllerKind: CaseIterable {
+        case globe
+        case plane
+        case crater
+        
+        /// A human-readable label for the camera controller kind.
+        var label: String {
+            switch self {
+            case .globe: return "Free pan round the globe"
+            case .plane: return "Orbit camera around plane"
+            case .crater: return "Orbit camera around crater"
+            }
+        }
+    }
+    
+    /// The camera controller of the scene view.
+    @State private var cameraController: CameraController {
+        didSet {
+            _ = GlobeCameraController()
+        }
+    }
+    
+    init() {
+        cameraController = Self.makeCameraController(kind: .crater)
+    }
```

Comment:
I think it is fine to default to the `GlobeCameraController` to make it simpler, which means…

```swift
@State private var cameraControllerKind: CameraControllerKind = .globe
    
@State private var cameraController: CameraController = GlobeCameraController()
```